config/admins.txt

```diff
@@ -7,7 +7,7 @@
 # NOTE: if the rank-name cannot be found in admin_ranks.txt, they will not be adminned! ~Carn #
 # NOTE: syntax was changed to allow hyphenation of ranknames, since spaces are stripped.      #
 ###############################################################################################
-Optimumtact = Host
+Optimumtact = Host\
```

Comment:
The question is why did you add a \?

anyway edit your .gitignore file

add config/admins.txt under *.backup.

you probably then want to add .gitignore under that so you don't push that change either.